### ISSUE TITLE
make test-modules fails

### ISSUE THREAD

**Issue by gambiteer** [gambit/gambit#860]
I get 
```
make test-modules
*** testing _define-library
*** all tests passed out of a total of 4 tests
*** testing _base64
*** all tests passed out of a total of 112 tests
*** testing _digest
*** all tests passed out of a total of 22 tests
*** not testing _geiser (lib/_geiser/test/test.scm missing)
*** not testing _git (lib/_git/test/test.scm missing)
*** testing _hamt
*** all tests passed out of a total of 119 tests
*** not testing _hamt/primitive-hamt (lib/_hamt/primitive-hamt/test/test.scm missing)
*** testing _http
*** all tests passed out of a total of 10 tests
*** testing _match
*** all tests passed out of a total of 21 tests
*** not testing _pkg (lib/_pkg/test/test.scm missing)
*** not testing _tar (lib/_tar/test/test.scm missing)
*** testing _test
"test.scm"@42.1: FAILED (_test-equal "\"test.scm\"@43.16: FAILED (test-assert (< 2 1)) GOT #f\n" (_test-msg (lambda () (test-assert (< 2 1))))) GOT "\"test.scm\"@43.16-43.37: FAILED (test-assert (< 2 1)) GOT #f\n"
*** 1 test failed after 4 tests
make[2]: *** [test-modules-pre] Error 1
make[1]: *** [test-modules] Error 1
make: *** [test-modules] Error 2
```

**Comment by gambiteer** [comment]
When I `make -i test-modules` that failure and
```
*** testing srfi/64
"test.scm"@42.1: FAILED (_test-equal "\"test.scm\"@43.16: FAILED (test-assert (< 2 1)) GOT #f\n" (_test-msg (lambda () (test-assert (< 2 1))))) GOT "\"test.scm\"@43.16-43.37: FAILED (test-assert (< 2 1)) GOT #f\n"
*** 1 test failed after 4 tests
make[3]: [test-modules-pre] Error 1 (ignored)
```
are the only ones I see,

**Comment by feeley** [comment]
Fixed by 584809890c5b2bb2113e0565c5f63c52cec79c1b.
